[jsk_robot/.travis.rosinstall] update .travis.rosinstall.kinetic/melodic/noetic to install naoqi_bridge source (kochigami-develop)

kochigami opened the pull request:
In https://github.com/jsk-ros-pkg/jsk_robot/pull/1674, I’d like to include naoqi_apps programs such as `/launch/tablet.launch, basic_awareness.launch, background_movement.launch` to `jsk_pepper_startup.launch`,
which belong to https://github.com/kochigami/naoqi_bridge/tree/kochigami-develop and are not merged to https://github.com/ros-naoqi/naoqi_bridge because merging process of the repository is suspended.

Because these packages are not cloned when running travis test, I got the following errors. 

In the case of kinetic:
```
 <testsuite errors="0" failures="1" name="roslaunch-check_jsk_pepper_startup__ROS_IP_10.68.0.2 NAO_IP_10.68.0.3 launch_dashboard_false.launch.xml" tests="1" time="1"><testcase classname="Results" name="test_ran" status="run" time="1"><failure message="roslaunch check [/github/home/ros/ws_jsk_robot/src/jsk_robot/jsk_naoqi_robot/jsk_pepper_startup/launch/jsk_pepper_startup.launch] failed" type="" /></testcase><system-out>&lt;![CDATA[
 
... 

[/github/home/ros/ws_jsk_robot/src/jsk_robot/jsk_naoqi_robot/jsk_pepper_startup/launch/jsk_pepper_startup.launch]:
  	while processing /opt/ros/kinetic/share/naoqi_apps/launch/tablet.launch:
  Invalid roslaunch XML syntax: [Errno 2] No such file or directory: u'/opt/ros/kinetic/share/naoqi_apps/launch/tablet.launch'
  ]]&gt;</system-out></testsuite>
```

I learned that naoqi_bridge release from https://github.com/ros-naoqi/naoqi_bridge-release stopped at kinetic.
However, in this case, I’d like to git clone https://github.com/kochigami/naoqi_bridge/tree/kochigami-develop, not https://github.com/ros-naoqi/naoqi_bridge.

I’m not familiar with travis, so if you have any comments, please let me know.
I apologize and thanks in advance.

pazeshun reviewed (changes requested):
Could you send a PR to https://github.com/ros-naoqi/naoqi_bridge and write its URL to the comments of .travis.rosinstall?
Also, changing only .travis.rosinstall instead of .travis.rosinstall.{ROS_DISTRO} is enough in this case, I think.
.travis.rosinstall is firstly used in every ROS_DISTRO, and .travis.rosinstall.{ROS_DISTRO} secondly adds distro-specific dependency.

kochigami commented:
Thank you very much for your help.

> Could you send a PR to https://github.com/ros-naoqi/naoqi_bridge and write its URL to the comments of .travis.rosinstall?

Thank you for your advice.
I sent PRs to naoqi_bridge and added comments to .travis.install.

basic_awareness: https://github.com/ros-naoqi/naoqi_bridge/pull/95
tablet: https://github.com/ros-naoqi/naoqi_bridge/pull/96
(background_awareness was already merged. I forgot this.)

> Also, changing only .travis.rosinstall instead of .travis.rosinstall.{ROS_DISTRO} is enough in this case, I think.
.travis.rosinstall is firstly used in every ROS_DISTRO, and .travis.rosinstall.{ROS_DISTRO} secondly adds distro-specific dependency.

Thank you for your advice.
I added change to only a .travis.rosinstall.

##### Memo (based on discussion with pazeshun)
- After my PRs are merged into https://github.com/ros-naoqi/naoqi_bridge, I should modify .travis.rosinstall like this:
```
# use naoqi_pose and naoqi_apps, wait for melodic release https://github.com/ros-naoqi/naoqi_bridge/issues/94.
- git: 
   uri: https://github.com/ros-naoqi/naoqi_bridge.git
```

- After naoqi_bridge is released for melodic/noetic, I should delete the change in .travis.rosinstall and add the following lines in each .travis.rosinstall.{ROS_DISTRO}.

.travis.rosinstall.kinetic => Source installation is required because naoqi_bridge is already released without updated naoqi_apps.
```
- git: # use naoqi_apps, which are updated after https://github.com/ros-naoqi/naoqi_bridge-release#naoqi_bridge-kinetic---055-0.
   uri: https://github.com/ros-naoqi/naoqi_bridge.git
```
.travis.rosinstall.melodic, noetic => Nothing is required if naoqi_bridge is released.

pazeshun commented on .travis.rosinstall:
```diff
@@ -17,3 +17,9 @@
+# use naoqi_apps, wait for merge https://github.com/ros-naoqi/naoqi_bridge/pull/95 & https://github.com/ros-naoqi/naoqi_bridge/pull/96
+# see https://github.com/jsk-ros-pkg/jsk_robot/pull/1718
+- git:
+   uri: https://github.com/kochigami/naoqi_bridge.git
```
Could you also specify `local-name`?

kochigami replied:
Thank you very much for your comment.
I specified local-name.
